server/src/main/java/org/apache/druid/indexing/overlord/supervisor/Supervisor.java

```diff
@@ -93,4 +94,9 @@ default Boolean isHealthy()
   LagStats computeLagStats();
 
   int getActiveTaskGroupsCount();
+
+  /**
+   * @return active base sequence names for reading and pending completion task groups of a seekable stream supervisor
```

Comment:
Update: prefixes instead of sequence names.

Reply:
Done